Source/common/SNTCommonEnums.h

```diff
@@ -129,6 +129,12 @@ typedef NS_ENUM(NSInteger, SNTSyncStatusType) {
   SNTSyncStatusTypeUnknown,
 };
 
+typedef NS_ENUM(NSInteger, SNTSyncContentEncoding) {
+  SNTSyncContentEncodingNone,
+  SNTSyncContentEncodingZlib,
```

Comment:
If we're going to key off the string `deflate` for the configuration value, I think it would be nice to also name the enum the same.

Reply:
Renamed to `SNTSyncContentEncodingDeflate`.